packages/site/.eleventy.js

```diff
@@ -4,7 +4,8 @@ module.exports = eleventyConfig => {
         'landing/dist': './',
         '../../packages/nimble-components/dist/storybook': 'storybook',
         '../../angular-workspace/dist/example-client-app': 'storybook/example-client-app',
-        '../../packages/nimble-blazor/dist/blazor-client-app': 'storybook/blazor-client-app'
+        '../../packages/nimble-blazor/dist/blazor-client-app': 'storybook/blazor-client-app',
+        '../../packages/nimble-components/performance': 'storybook/wafer-performance'
```

Comment:
```suggestion
        '../../packages/nimble-components/performance': 'storybook/performance/wafer-map'
```

And also update the main storybook page to have a link to it: https://github.com/ni/nimble/blob/b61e3164866c8703fad31b2673fc88ebb694c6fd/packages/nimble-components/src/tests/nimble-intro.stories.mdx#L21